packages/vue/src/components/list/SingleDropdownList.jsx

```diff
@@ -1,14 +1,14 @@
 import { Actions, helper } from '@appbaseio/reactivecore';
 import VueTypes from 'vue-types';
+import { isEqual } from '@appbaseio/reactivecore/lib/utils/helper';
```

Comment:
Why did you import it i.e where you're using it?
It will have issues in watch mode if you import it like that.

Reply:
I didn't import it, re-order it to solve `eslint` warning of `absolute imports should be before relative imports`